README.rst

```diff
@@ -1,8 +1,15 @@
 BrowserID Page Object Model
 ===========================
 
-Selenium compatible page object model for
-`Mozilla Persona <https://login.persona.org/>`_ (previously known as BrowserID).
+This repository is no longer actively maintained
+------------------------------------------------
+
+Due to the shutdown of the Mozilla Persona service, this supporting project is no longer being used nor actively maintained.
+
+Documentation may be found `here <https://github.com/mozilla/bidpom/wiki>`_.
```

Comment:
We have a link to the documentation at the bottom of the README, so I don't think we need it twice. I won't block the review though, so feel free to merge or push a change before doing so.